packages/summer-earn-protocol-subgraph/src/mappings/harborCommand.ts

```
@@ -1,4 +1,4 @@
-import { Address, BigInt, ethereum, log } from '@graphprotocol/graph-ts'
+import { Address, BigInt, dataSource, ethereum, log } from '@graphprotocol/graph-ts'
```

Biome (2.1.2)

[error] Do not shadow the global "BigInt" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)